script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -7,46 +7,46 @@ I don't want to go through something like that ever again.
 If I stay with Arcueid, the same thing will definitely happen again. If I don't want that, I have no choice but to go back to the mansion.
 <Page3>
 "――――――"
-...There's a big flaw in my choice.
+...There's a big flaw in my choice.//Maybe revisit
```

Comment:
Reads awkwardly to me but the JP doesn't give much to work with. 

> 自分の選択には大きな欠陥がある。

My read is that he knows he's making a bad decision?

Reply:
_little_ bit different, but reads more naturally.
...There's a problem with that choice though.

Reply:
Yeah, the other thing that felt natural was 'flaw in my reasoning' but it felt like a stretch